RelationalAI.Test/DatabaseTest.cs

```diff
@@ -25,11 +25,7 @@ public async Task DatabaseTest()
 
             await Assert.ThrowsAsync<HttpError>(async () => await client.DeleteDatabaseAsync(Dbname));
 
```

Comment:
To insure we're backwards compatible, we should not modify any existing tests, only add new tests for new functionality.